update-bot/wlmbots/lib/template_checker.py

```diff
@@ -11,6 +11,7 @@ def __init__(self, config=None):
             config = {}
         self.tpl_match_regex = None
         self.config = config
+        self._config = config
```

Comment:
Did pylint complain here? Was the complaint that the internal field self._config was not initialized in the constructor? That double assignment looks weird to me. Maybe you can change it to 
```python
self._config = None
self.config = config
```